crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Performance improvement when sending validation receipts. When a batch of DHT ops is being processed and an author is unreachable while attempting to send a validation receipt, all the other receipts for that
+  author will be skipped. The network send already has a retry built in so when an author was offline and a batch of `N` receipts were ready to be sent it would take `N` minutes for the workflow to work through
+  them. This was unfair to other nodes that are still online because they would have to wait unreasonably long to get their validation receipts.
+- Resilience improvement with handling keystore errors in the validation receipt workflow. Previously, all errors caused the workflow to restart from the beginning. This was good for transient errors such as the
+  keystore being unavailable but it also meant that a single validation receipt failing to be signed (e.g. due to a local agent key being removed from the keystore) would prevent any more validation receipts being
+  sent by that conductor.
+- **BREAKING CHANGE** Addressed an outstanding technical debt item to make the validation receipt workflow send a network notification (fire and forget) rather than waiting for a response. When the validation receipt
+  workflow was written this functionality wasn't available but now that it is, sending validation receipts can be sped up by not waiting for a peer to respond. Note that the conductor will still accept and respond
+  to validation receipt requests so that older conductors can send validation receipts to newer ones. However, older conductors will reject validation receipt notifications from newer conductors. This is labelled as
+  a breaking change but really the break will be for older conductors, and newer conductors should work as expected when talking to a mixed network.
```

Comment:
mmm if old nodes break, is there any reason to make newer nodes support a mixed network?

Reply:
I still wonder if we should implement a hard protocol version somewhere that we can bump and prevent differing node versions from speaking to each other when that version is bumped.

Reply:
> mmm if old nodes break, is there any reason to make newer nodes support a mixed network?

I guess just so they at least keep running?

Reply:
There's no crash condition I don't think, if a wrong message is sent it will just respond with an error. 

The only benefit on 0.2 is that people who upgrade first will be able to receive validation receipts from an existing conductor. I don't know how important that really is because nodes should be able to sync existing data without validation receipts.

It's a fair question whether we should keep this. It feels like a smaller break for 0.2 like this but only by a small amount. I think it should immediately go on `develop` after this change if we do keep it.

Reply:
Again as above, this issue has been forced and is now a full breaking change between conductor versions